src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,54 @@
 
 #include "t06_homework.h"
 #include <iostream>
-
+#include <set>
+#include <string>
 using namespace std;
 
 int t06_homework() {
-
+string a,c,f;
+    set<string> b;
+    set<string> k;
+    int N,e=0,g=1;
+    cin>>N;
+    for(int i=0;i<N;i++){
+        cin>>a;
+        b.insert(a);
+        for(int j=0;j<a.length();j++) {
+            if (a[j] >= 65 && a[j] <= 90)
+                a[j] += 32;
+        }
+            k.insert(a);
+    }
+    cin.ignore();
+    getline(cin,c);
+            for (int i = 0; i <= c.size(); i++) {
+                if (c[i] !=' '&&i!=c.size())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [When i==c.size(), c[i] is out of bounds.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4552363)